crates/analytics/src/search.rs

```diff
@@ -26,23 +26,47 @@ pub async fn msearch_results(
         .execute(query_builder)
         .await
         .change_context(OpenSearchError::ConnectionError)?
-        .json::<OpenMsearchOutput<Value>>()
+        .json::<OpenMsearchOutput>()
         .await
         .change_context(OpenSearchError::ResponseError)?;
 
     Ok(response_body
         .responses
         .into_iter()
         .zip(SearchIndex::iter())
-        .map(|(index_hit, index)| GetSearchResponse {
-            count: index_hit.hits.total.value,
-            index,
-            hits: index_hit
-                .hits
-                .hits
-                .into_iter()
-                .map(|hit| hit._source)
-                .collect(),
+        .map(|(index_hit, index)| match index_hit {
+            OpensearchOutput::Success(success) => {
+                if success.status == 200 {
+                    GetSearchResponse {
+                        count: success.hits.total.value,
+                        index,
+                        hits: success
+                            .hits
+                            .hits
+                            .into_iter()
+                            .map(|hit| hit._source)
```

Comment:
```suggestion
                            .map(|hit| hit.source)
```

The `_` prefix is a rust convention for unused variables can we remove it?

Reply:
Removed it